### ISSUE TITLE
[question] Pretraining problem

### ISSUE THREAD

**Issue by HJ-TANG** [hill-a/stable-baselines#973]
I've got a relatively good policy on my own environment with the pretraining. After pretraining, I load the pre-trained model and test it in the following way:

```python
model = SAC.load("pretrained_models_sac")
episode_reward = 0
obs = env.reset()

while True:
    action, _states = model.predict(obs)
    obs, rewards, dones, info = env.step(smoothed_action)
    episode_reward += rewards

    if dones:
        break
```

The `episode_reward` can always be more than `3000` which meets my expectations.

But when I try to train the pre-trained model with SAC like this (the `learning_starts` is set to a large value because I want it generates more trajectories before starting learning):

```python
model = SAC.load("pretrain_models_sac", env=env)
model.learning_starts = int(1e5)
model.learn(total_timesteps=total_steps)
model.save("model01")
```

the log before starting learning is like (the `total timesteps` is smaller than `learning_starts` param so I think the learning has not started?):

```
--------------------------------------
| current_lr              | 0.0003   |
| episodes                | 220      |
| fps                     | 2271     |
| mean 100 episode reward | 56.9     |
| n_updates               | 0        |
| success rate            | 0        |
| time_elapsed            | 10       |
| total timesteps         | 23190    |
--------------------------------------
```

The `mean 100 episode reward` is only about `60`.  

In my understanding, learning has not started so the params are supposed to be the same as these stored in the pre-trained model. And then the `mean 100 episode reward` should not be such low.

I'm wondering if there is anything wrong with my way using the pretraining?

**Comment by araffin** [comment]
Hello,
You may find an answer here ;)
https://github.com/hill-a/stable-baselines/issues/929#issuecomment-656562248

Testing is different from initial exploration and here `learning_starts` sampling action randomly from the action space.

**Comment by HJ-TANG** [comment]
> Hello,
> You may find an answer here ;)
> [#929 (comment)](https://github.com/hill-a/stable-baselines/issues/929#issuecomment-656562248)
> 
> Testing is different from initial exploration and here `learning_starts` sampling action randomly from the action space.

Oh, I misunderstood the `learning_starts` here...

Thanks for your quick reply! @araffin I'll close this issue.
